config/resources/domain.json

```diff
@@ -41,6 +41,12 @@
           "target": "roadSignConcept",
           "cardinality": "many"
         },
+        "mainSigns": {
+          "predicate": "lblodmow:heeftOnderbordConcept",
+          "target": "roadSignConcept",
+          "cardinality": "many",
+          "inverse": true
```

Comment:
I think this would be a direct relation or include another relation to putting the child signs as direct relation.

Reply:
Ah no, sorry, it's in the subsigns. 😅

Reply:
Wait, but the main sign is the sign who has the sub signs, no?

Reply:
Indeed, the main sign has subsigns.
This addition is the reverse relation - i.e. to get all the possible main signs for a given subsign.

Reply:
Ok, I got it.